### PR TITLE
community: add the integrator code of conduct

### PR DESCRIPTION
**Pull request by monperrus**
The great Spoon community keeps growing, and we need to explain how to recognize a top-contributor as integrator.

What do you think of this?

### CODE REVIEW

**Comment by surli** [comment]
It's ok for me.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+The integrator code of conduct
+------------------------------
+
+The integrators are the developers who have write access to the repository. In particular, they can merge pull requests to the `master` branch. They shall respect the following rules:
+
+* One never pushes directly to `master`
+* One only merges atomic pull requests (single bug fix or single feature)
+* One only merges well tested and well documented pull requests, after thorough code review
+* One never merges one's own pull request
+* One leaves pull requests opened at least 1 day before merging, so that the community is aware and can comment on them
```
Comment (tdurieux) [inline CONTRIBUTING.md]: add a link to the commit message convention: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#type

**Comment by tdurieux** [comment]
Ok for me

**Comment by surli** [comment]
@danglotb do you agree?

**Comment by danglotb** [comment]
I do

**Comment by monperrus** [comment]
all integrators agree, we can proceed with merge @surli?

**Pull request merged**